components/SingleRatingsTable/index.tsx

```diff
@@ -0,0 +1,381 @@
+import React, { useCallback, useMemo } from 'react';
```

Comment:
differences from normal RatingsTable
- No link to self
- Initial filter to season